nvidia-bert/ort_addon/ort_supplement/ort_supplement.py

```diff
@@ -112,13 +112,13 @@ def map_optimizer_attributes(name):
     model = ORTTrainer(model, None, bert_model_description(args), "LambOptimizer", 
         map_optimizer_attributes,
         IODescription('Learning_Rate', [1,], torch.float32),
-        device, postprocess_model=postprocess_model, 
+        device, 
         gradient_accumulation_steps=args.gradient_accumulation_steps,
         world_rank=args.world_rank, world_size=args.world_size,
         use_mixed_precision = True if args.fp16 else False,
         allreduce_post_accumulation = True if args.allreduce_post_accumulation else False,
         partition_optimizer = True if args.partition_optimizer else False,
```

Comment:
>partition_optimizer [](start = 8, length = 19)

renamed to " deepspeed_zero_stage" - [https://github.com/microsoft/onnxruntime/commit/a241eb0bbe44636c8656af17001470c39346f9c5#diff-04d66baee289e592c46ebcbf35e3a836](https://github.com/microsoft/onnxruntime/commit/a241eb0bbe44636c8656af17001470c39346f9c5#diff-04d66baee289e592c46ebcbf35e3a836)

Reply:
Ah, good catch 👍